Add rate limit section to API (#5225)
PostHog/posthog.com: contents/docs/api/index.mdx

You can then just call the `"next"` URL to get the next set of results.

> ## Rate Limiting

Reviewer: ```suggestion
## Rate limiting
```

All requests to PostHog, except for evaluating feature flags and sending events (i.e. the [POST-only public endpoints](/docs/api/post-only-endpoints)), are rate limited.

There's separate limits for different kinds of resources.

For all CRUD endpoints, the rate limits are `480/minute` and `4800/hour`. 

> For all analytics endpoints (like calculating insights, retrieving persons, retrieving session recordings), the rate limits are `240/minute` and `1200/hour`.

Reviewer: ```suggestion
For all analytics endpoints (such as calculating insights, retrieving persons, retrieving session recordings), the rate limits are `240/minute` and `1200/hour`.
```

Nit: These are examples of endpoints, not things which are similar to endpoints.

> Note that these limits apply to the entire team, so if you have a script requesting persons that hits the rate limit, using the PostHog UI to get persons would get rate limited as well.

Author: Makes sense, is possible!

> Note that these limits apply to the entire team, so if you have a script requesting persons that hits the rate limit, using the PostHog UI to get persons would get rate limited as well.

Reviewer 2: ```suggestion
Note that these limits apply to the entire team, so if you have a script requesting persons that hits the rate limit, using the PostHog UI to get persons would get rate limited as well.

For large or regular exports of events we would recommend [using export apps](https://posthog.com/docs/migrate/export-events).

If you are wanting to use the PostHog API beyond these limits (for example using PostHog to power your user-facing dashboards) feel free to email us at [customers@posthog.com](to:customers@posthog.com).
```

Reviewer: ```suggestion
Note that these limits apply to the entire team, so if you have a person-requesting script that hits the rate limit, using the PostHog UI to get persons would get rate limited as well.

For large or regular exports of events we would recommend [using export apps](https://posthog.com/docs/migrate/export-events).

If want to use the PostHog API beyond these limits (for example by using PostHog to power your user-facing dashboards) then please email us at [customers@posthog.com](to:customers@posthog.com).
```

Just removes some passive voice from Luke's suggestion

> Note that these limits apply to the entire team, so if you have a script requesting persons that hits the rate limit, using the PostHog UI to get persons would get rate limited as well.

Reviewer: ```suggestion
Note that these limits apply to the entire team, so if you have a person-requesting script that hits the rate limit, using the PostHog UI to get persons would get rate limited as well.
```
This just parses a little simpler IMHO, even if the language is a bit made-up 🤷
